### ISSUE TITLE
Build error on Xcode 7.3 beta 2 (Swift 2.2)

### ISSUE THREAD

**Issue by norio-nomura** [realm/SwiftLint#425]
log:

>SwiftLint/Source/SwiftLintFrameworkTests/IntegrationTests.swift:29:54: error: cannot convert value of type 'String' to expected argument type 'StaticString'
>            XCTFail($0.reason, file: $0.location.file!, line: UInt($0.location.line!))

That is [expanding linting result into `XCTFail()`](https://github.com/realm/SwiftLint/blob/0.7.1/Source%2FSwiftLintFrameworkTests%2FIntegrationTests.swift#L28-L30) that introduced on https://github.com/realm/SwiftLint/pull/404:
```swift
class IntegrationTests: XCTestCase {
    func testSwiftLintLints() {
        // This is as close as we're ever going to get to a self-hosting linter.
        let directory = (((__FILE__ as NSString)
            .stringByDeletingLastPathComponent as NSString)
            .stringByDeletingLastPathComponent as NSString)
            .stringByDeletingLastPathComponent
        NSFileManager.defaultManager().changeCurrentDirectoryPath(directory)
        let config = Configuration(path: Configuration.fileName)
        let swiftFiles = config.lintableFilesForPath("")
        XCTAssert(swiftFiles.map({$0.path!}).contains(__FILE__), "current file should be included")
        let violations = swiftFiles.flatMap {
            Linter(file: $0, configuration: config).styleViolations
        }
        violations.forEach {
            XCTFail($0.reason, file: $0.location.file!, line: UInt($0.location.line!))
        }
    }
}
```
Related:
https://bugs.swift.org/browse/SR-308?jql=text%20~%20%22staticstring%22
https://github.com/apple/swift/pull/888
https://github.com/apple/swift-corelibs-xctest/pull/27

:cry:

**Comment by NachoSoto** [comment]
Yeah, I opened https://github.com/Quick/Nimble/pull/243 in Nimble with the same issue.
